src/game/client/components/chat.cpp

```diff
@@ -1353,6 +1353,12 @@ void CChat::HandleCommands(float x, float y, float w)
 			NextActiveCommand(&m_CommandStart);
 		}
 
+		if(!DisplayCount)
+		{
+			m_SelectedCommand = -1;
+			return;
```

Comment:
This `return` is not useful, because the rest of the function is in a `if(DisplayCount > 0)`. This should probably just be a if... else actually.